docs/detections/prebuilt-rules/downloadable-packages/0-16-2/prebuilt-rule-0-16-2-microsoft-build-engine-started-an-unusual-process.asciidoc

```diff
@@ -0,0 +1,80 @@
+[[prebuilt-rule-0-16-2-microsoft-build-engine-started-an-unusual-process]]
+=== Microsoft Build Engine Started an Unusual Process
+
+An instance of MSBuild, the Microsoft Build Engine, started a PowerShell script or the Visual C# Command Line Compiler. This technique is sometimes used to deploy a malicious payload using the Build Engine.
```

Comment:
```suggestion
An instance of the Microsoft Build Engine (MSBuild) started a PowerShell script or the Visual C# command line compiler. This technique is sometimes used to deploy a malicious payload using MSBuild.
```

Reply:
Unsure if the last edit is correct, so feel free to ignore it and go with the original if it's not.